Gemfile

```diff
@@ -5,7 +5,9 @@ source 'http://rubygems.org'
 gemspec
 
 gem 'appraisal'
+gem 'appraisal-matrix'
 gem 'bump', '~> 0.6.1'
+gem 'mutex_m'
```

Comment:
I'm curious why we need to be explicit with `mutex_m`? Isn't that implicitly used by one of the other gems like `appraisal-matrix`?

Reply:
In Ruby 3.3, it seems we need to explicitly add it to the Gemfile (specifically for the Appraisal gemfiles):
https://github.com/Invoca/process_settings/actions/runs/9882625967/job/27295876025

> /home/runner/work/process_settings/process_settings/vendor/bundle/ruby/3.3.0/gems/activesupport-6.0.6.1/lib/active_support/notifications/fanout.rb:3: warning: mutex_m was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add mutex_m to your Gemfile or gemspec.